File: sample/OrderService.cs
namespace App.Sample;

using System.Collections.Generic;

/// <summary>class FakeOne { } — 문서 주석은 코드가 아니다.</summary>
public sealed partial class OrderService
{
    // struct FakeTwo { }
    private const string LogPath = @"C:\temp\logs\interface FakeThree { }";

    private const string Json = """
        { "kind": "enum FakeFour { }" }
        """;

    private readonly List<string> _names = new();

    public string Describe(string name) => $"{name} record FakeFive";
}

File: sample/OrderService.swift
import Foundation

/* 바깥 주석 /* 안쪽에 struct FakeOne { } */ 아직 주석이다 class FakeTwo { } */
final class OrderService {

    // protocol FakeThree { }
    private let pattern = #"enum FakeFour { case a }"#

    private let sql = """
        actor FakeFive { }
        """

    class func make() -> OrderService {
        OrderService()
    }

    func describe(_ name: String) -> String {
        "\(name) struct FakeSix { }"
    }
}
